test/dotnetaspire/dotnet_env.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+export DOTNET_NOLOGO=true
+export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true
+export DOTNET_GENERATE_ASPNET_CERTIFICATE=false
```

Comment:
We need to configure the dev cert.

Reply:
See https://github.com/dotnet/aspire/blob/a798b766c1bab0afc2e8ac0dad0e92c6283e60fd/.devcontainer/dogfooding-nightly/devcontainer.json#L31

Reply:
It's not needed for tests unless I launch something, right?

Reply:
As for the install -- feels like if I add that it should be in the "dotnet" feature...?

Reply:
what do you mean?

Reply:
the "dotnet" features installs the SDK/ASP.NET. if any "feature" is going to trust the dev certs it's going to be the one that installs them, not the "dotnetaspire" feature that just chains it.

I will put up a PR with that.

Reply:
offline we decided this isn't appropriate for a container situation, the user should do it